python/neuroglancer/ingest/lib.py

```diff
@@ -181,9 +183,11 @@ def download_from_gcloud(cloudpaths, destination, log=None, gzip=False):
   # so we're using streaming mode (-I) to enable it to handle arbitrary numbers of files
   # -m = multithreaded upload
 
-  gsutil_upload_cmd = "gsutil -m cp {logging} -I {destination}".format(
+  gsutil_upload_cmd = "gsutil {multiprocessing} {quiet} cp {logging} -c -I {destination}".format(
```

Comment:
-m) We want to disable multiprocessing in the cloud because we already fully load all CPUs as much as we can (e.g. I use 1.5 pods per CPU). Multiprocessing means the pods interfere with each other.

-c) When processing the outer regions of a dataset, often a particular chunk is not present at all. -c means that gsutil will continue to process all requested files in a batch, without it it dies and you get a block of chunks in a black sea. -m enables -c implicitly, so this is only a problem when CLOUD_COMPUTING=1

-q) In the cloud, file download reports swamp useful information, so we disable it when CLOUD_COMPUTING is enabled.